.gitignore

```diff
@@ -10,6 +11,8 @@ pytest.xml
 super-linter-output
 super-linter.log
 
+# REMOVE THIS FROM SRC
```

Comment:
Would you be able to go over what this means?  Should these two lines be deleted?